frontend/src/components/AddConcepts.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useState, useEffect} from 'react';
+import DataTable, {createTheme} from "react-data-table-component";
+
+import {useDataGetter, DataGetter} from "../state/DataGetter";
+
+interface Concept {
+  readonly concept_id: number;
+  readonly concept_name: string;
+  readonly domain_id: string;
+  readonly vocabulary_id: string;
+  readonly concept_class_id: string;
+  readonly standard_concept: string;
+  readonly concept_code: string;
+  readonly invalid_reason: string;
+  readonly domain_cnt: number;
+  readonly domain: string;
+  readonly total_cnt: number;
+  readonly distinct_person_cnt: string;
+}
+
+const columns = [
+  {
+    name: "Concept Name",
```

Comment:
## Display full concept name
Right now if the concept name is too large, it gets truncated (...).

<img width="486" alt="Screenshot 2024-04-26 at 6 46 51 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/bfd98c7f-f6a3-476b-8347-21a5a7ac8d32">

Perhaps when we add the "add concept" UI, then when the user selects the concept row, we can have it show the full name and that will be sufficient?

In addition, we could add an `onHover` tooltip as well, but maybe the former is sufficient.

Reply:
I should just wrap text like happens in the comparison table